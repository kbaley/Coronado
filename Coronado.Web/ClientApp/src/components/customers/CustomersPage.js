import React, { Component } from 'react';
import NewCustomer from './NewCustomer';

export class CustomersPage extends Component {
  render() {
    return (
      <div>
        <h1>
          Customers <NewCustomer />
        </h1>
      </div>
    );
  }
}

export default CustomersPage;