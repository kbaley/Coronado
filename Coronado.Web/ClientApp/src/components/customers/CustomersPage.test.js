import React from 'react';
import CustomersPage from "./CustomersPage";
import renderer from "react-test-renderer";

jest.mock('./NewCustomer', () => 'new customer');

describe('CustomersPage tests', () => {
  it('should render the component', () => {
    const tree = renderer
      .create(<CustomersPage />);
    expect(tree.toJSON()).toMatchSnapshot();
  });

});