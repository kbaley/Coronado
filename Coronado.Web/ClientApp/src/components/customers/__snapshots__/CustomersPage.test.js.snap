// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`CustomersPage tests should render the component 1`] = `
<div>
  <h1>
    Customers 
    <new customer />
  </h1>
</div>
`;
